src/components/BaseStyles/BaseStylesService.ts

```diff
@@ -212,7 +212,6 @@ export const createBaseStyles = ({ theme }: StyleProps) => css`
     font-feature-settings: 'kern';
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
-    overflow-x: hidden;
```

Comment:
I did deal with this one a bit more in my personal project and found that this alone actually introduces a problem on mobile (i.e., the page ends up not fitting on the screen and there is horizontal scroll). What I then figured is that it's fine to put `overflow-x: hidden;` exclusively on the `html` tag. We can move the overflow up to the sections between lines `185` and `191`, I guess.

Reply:
Interesting, do you know which element/component caused it to overflow?

Good to know that the `overflow` rule on the `html` element doesn't have the same effect on its children.